lib/members_page.rb

```diff
@@ -0,0 +1,69 @@
+#!/bin/env ruby
+# encoding: utf-8
+# frozen_string_literal: true
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::AbsoluteUrls
+
+  field :member_rows do
```

Comment:
Ah, the new `Metrics/BlockLength` should take of this, although it's set by default to 25, even though the default for methods is 10!   I've changed our default for blocks to 10 as well in https://github.com/everypolitician/everypolitician-data/pull/24540

(Interestingly, there doesn't seem to be an equivalent to the method Complexity metrics — we might have to add our own for that..